homeassistant/components/http/auth.py

```diff
@@ -86,9 +104,27 @@ def async_user_not_allowed_do_auth(
     return "User cannot authenticate remotely"
 
 
-@callback
-def setup_auth(hass: HomeAssistant, app: Application) -> None:
+async def setup_auth(hass: HomeAssistant, app: Application) -> None:
```

Comment:
Perhaps rename method to `async_setup_auth` to explicitly reflect change to coroutine.